src/main/java/com/group8/library_management/repository/BorrowRecordRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.group8.library_management.repository;
+
+import com.group8.library_management.entity.BorrowRecord;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface BorrowRecordRepository extends JpaRepository<BorrowRecord, Long> {
+    List<BorrowRecord> findByUser_Id(Integer userId);
```

Comment:
```suggestion
    List<BorrowRecord> findByUserId(Integer userId);
```
trong java ko dùng `_` nha
Mình theo coding convention là `camelCase`

Reply:
Em đã sửa rồi ạ